pandas/core/indexes/base.py

```diff
@@ -5284,11 +5269,11 @@ def get_slice_bound(self, label, side: str_t, kind) -> int:
                 slc = lib.maybe_indices_to_slice(
                     slc.astype(np.intp, copy=False), len(self)
                 )
-            if isinstance(slc, np.ndarray):
-                raise KeyError(
-                    f"Cannot get {side} slice bound for non-unique "
-                    f"label: {repr(original_label)}"
-                )
```

Comment:
without this indentation we end up doing an unnecessary duplicate check

Reply:
true, I'll revert this one